soc/st/stm32/stm32n6x/CMakeLists.txt

```diff
@@ -8,3 +8,20 @@ zephyr_sources(
 zephyr_include_directories(.)
 
 set(SOC_LINKER_SCRIPT ${ZEPHYR_BASE}/include/zephyr/arch/arm/cortex_m/scripts/linker.ld CACHE INTERNAL "")
+
+if(${CMAKE_HOST_SYSTEM_NAME} STREQUAL Windows)
+  set(SIGNING_TOOL STM32_SigningTool_CLI.exe)
+else()
+  set(SIGNING_TOOL STM32_SigningTool_CLI)
+endif()
+
+set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
+  COMMAND ${SIGNING_TOOL}
+  -in ${PROJECT_BINARY_DIR}/${CONFIG_KERNEL_BIN_NAME}.bin
+  -nk -of 0x80000000 -t fsbl -hv 2.3
```

Comment:
address should come from dts using cmake functions

Reply:
-of is for option flags. It is in fact not needed -> removed.